### PR TITLE
[WIP][3.3.5][6.x] Scripts/Karazhan: Clean up instance script

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Kittnz** [comment]
:+1:

File: src/server/scripts/EasternKingdoms/Karazhan/karazhan.h [hunk 1]
```diff
@@ -23,25 +23,22 @@
...
-    TYPE_OPTIONAL_BOSS              = 4,
-    TYPE_OPERA                      = 5,
-    TYPE_CURATOR                    = 6,
-    TYPE_ARAN                       = 7,
-    TYPE_TERESTIAN                  = 8,
-    TYPE_NETHERSPITE                = 9,
-    TYPE_CHESS                      = 10,
-    TYPE_MALCHEZZAR                 = 11,
-    TYPE_NIGHTBANE                  = 12,
+    DATA_ATTUMEN                    = 1,
```
Comment (ccrs) [inline src/server/scripts/EasternKingdoms/Karazhan/karazhan.h]: start from 0

**Comment by Treeston** [comment]
:+1:

File: src/server/scripts/EasternKingdoms/Karazhan/karazhan.h (hunk 1 above)
Comment (tkrokli) [inline src/server/scripts/EasternKingdoms/Karazhan/karazhan.h]: That would have been the right thing to do, if it had been done by the original scripter. Are you sure that the values are only counters and can be set to start from 0 in the scripts they are used in? (I don't know, it just feels edgy.)

**Comment by Nyeriah** [comment]
https://github.com/TrinityCore/TrinityCore/commit/f9eff81388b12f6f8275f892c67bf197188c3450

**Pull request closed without merging**